DemoEdgeApp/ModuleWrapper/MockMessageInjector.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace ModuleWrapper
+{
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Hosting;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+    public delegate Task InjectMessageAsyncDelegate(IConfiguration configuration, IModuleClient moduleClient);
+    public class MockMessageInjector : IHostedService
+    {
+        public IConfiguration Configuration { get; }
+        public IModuleClient ModuleClient { get; }
+        public InjectMessageAsyncDelegate InjectMessage { get; }
+        public MockMessageInjector(IConfiguration configuration,
+            IModuleClient moduleClient,
+            InjectMessageAsyncDelegate injectMessage)
+        {
+            Configuration = configuration;
+            ModuleClient = moduleClient;
+            InjectMessage = injectMessage;
+        }
+        public async Task StartAsync(CancellationToken cancellationToken)
+        {
+            // TODO: Make this repeating
```

Comment:
Is this reminder for a future iteration? Or did you mean to implement this now?

Reply:
maybe I should remove the Mock Message code file, as they are migrated from the influxDB sample code that I make reference when I integrate the influxDB to reliable relay solution. The Mock Message relevant parts are not functionable in this solution.